File: exs/mundo_1/ruby/033.rb
=begin
Desafio 033

Problema: Faça um programa que leia três números e mostre qual é o maior e qual é o menor.

Resolução do problema:
=end

print'Digite o primeiro número: '
num1 = gets.chomp.to_f

print'Digite o segundo número: '
num2 = gets.chomp.to_f

print'Digite o terceiro número: '
num3 = gets.chomp.to_f

maior = num1
menor = num1

if num2 > num1 and num2 > num3
    maior = num2
end

if num3 > num1 and num3 > num2
    maior = num3
end

if num2 < num1 and num2 < num3
    menor = num2
end

if num3 < num1 and num3 < num2
    menor = num3
end

puts"O número #{maior} é o MAIOR!"
puts"O número #{menor} é o MENOR!"

File: exs/mundo_1/golang/033.go
/*
Desafio 033

Problema: Faça um programa que leia três números e mostre qual é o maior e qual é o menor.

Resolução do problema:
*/

package main

import (
	"fmt"
)

var num1, num2, num3 int

func main(){
	fmt.Print("Digite o valor de num1: ")
	fmt.Scanln(&num1)
	fmt.Print("Digite o valor de num2: ")
	fmt.Scanln(&num2)
	fmt.Print("Digite o valor de num3: ")
	fmt.Scanln(&num3)

	maior := num1
	menor := num1

	if num2 > num1 && num2 > num3{
    	maior = num2
	}
	if num3 > num1 && num3 > num2{
    	maior = num3
	}


	if num2 < num1 && num2 < num3{
		menor = num2
	}
	if num3 < num1 && num3 < num2{
		menor = num3
	}

	fmt.Printf("O número %v é o maior.\n", maior)	
	fmt.Printf("O número %v é o menor.\n", menor)	
}
